frameit/bin/frameit

```diff
@@ -19,6 +19,7 @@ class FrameItApplication
     program :help_formatter, :compact
 
     global_option('--verbose') { $verbose = true }
+    FastlaneCore::CommanderGenerator.new.generate(Frameit::Options.available_options)
```

Comment:
frameit previously had no `ConfigItem` options, but now it has one. This line is required to turn the `ConfigItem`s into `global_option`s in the `Commander` context.